<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMikPPPManager</class>
 <widget class="QMainWindow" name="QMikPPPManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMikPPPManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="connectButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Configuración</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="apiUsersButton">
         <property name="text">
          <string>Usuarios API</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pppLogsButton">
         <property name="text">
          <string>Registros</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="connectionConfigButton">
         <property name="text">
          <string>Routers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="advancedConfigButton">
         <property name="text">
          <string>Avanzada</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="localConfigButton">
         <property name="text">
          <string>Apariencia</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pppProfilesButton">
         <property name="text">
          <string>Perfiles PPP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="schedulerButton">
         <property name="text">
          <string>Programación</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="6">
     <widget class="QROSSecretTableWidget" name="usersTable"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="addUserButton">
      <property name="text">
       <string>Añadir usuario</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>181</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="filterGroupBox">
      <property name="title">
       <string>Filtro</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="textFilterLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="fieldFilterComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="serviceStateFilterComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="VoIPFilterComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="portFilterComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>180</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="disconnectButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Desconectar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QROSSecretTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Widgets/QROSSecretTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>usersTable</tabstop>
  <tabstop>filterGroupBox</tabstop>
  <tabstop>textFilterLineEdit</tabstop>
  <tabstop>fieldFilterComboBox</tabstop>
  <tabstop>serviceStateFilterComboBox</tabstop>
  <tabstop>VoIPFilterComboBox</tabstop>
  <tabstop>portFilterComboBox</tabstop>
  <tabstop>addUserButton</tabstop>
  <tabstop>localConfigButton</tabstop>
  <tabstop>connectionConfigButton</tabstop>
  <tabstop>apiUsersButton</tabstop>
  <tabstop>advancedConfigButton</tabstop>
  <tabstop>pppProfilesButton</tabstop>
  <tabstop>pppLogsButton</tabstop>
  <tabstop>schedulerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
